pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -315,24 +372,37 @@ func (infc *IngNodeFwController) GetBPFMapContentForTest() (map[BpfLpmIpKeySt]Bp
 func (infc *IngNodeFwController) Close() error {
 	var errors []error
 
-	klog.Info("Removing all pins")
-	if err := infc.removeAllPins(); err != nil {
-		errors = append(errors, fmt.Errorf("could not remove all eBPF pins, err: %q", err))
+	ebpfManager := false
+	ebpfManagerMode, ok := os.LookupEnv(ebpfProgramMangerEnvVar)
+	if ok {
+		val, err := strconv.ParseBool(ebpfManagerMode)
+		if err != nil {
+			return fmt.Errorf("failed to convert %q to bool: %v", ebpfManagerMode, err)
+		}
+		ebpfManager = val
 	}
+	klog.Infof("Check bpfman manager mode %v", ebpfManager)
```

Comment:
This doesn't seem to check anything and seems to be more of a statement:

```suggestion
	klog.Infof("eBPF manager mode is set to: %v", ebpfManager)
```